<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DriveWidget</class>
 <widget class="QFrame" name="DriveWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>40</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>0</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ActionsContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="frameShape">
   <enum>QFrame::Panel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QLabel" name="driveLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>26</width>
       <height>26</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>26</width>
       <height>26</height>
      </size>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>104</red>
           <green>104</green>
           <blue>104</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>104</red>
           <green>104</green>
           <blue>104</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>120</red>
           <green>120</green>
           <blue>120</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>120</red>
           <green>120</green>
           <blue>120</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="text">
      <string>1:</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="margin">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="buttonMountDisk">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>28</width>
       <height>28</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>28</width>
       <height>28</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="buttonMountFolder">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>28</width>
       <height>28</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>28</width>
       <height>28</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="buttonSave">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>28</width>
       <height>28</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>28</width>
       <height>28</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="autoSave">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>28</width>
       <height>28</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>28</width>
       <height>28</height>
      </size>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="autoRaise">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="buttonEditDisk">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>28</width>
       <height>28</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>28</width>
       <height>28</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="buttonToggleChip">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>21</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>21</width>
       <height>25</height>
      </size>
     </property>
     <property name="popupMode">
      <enum>QToolButton::DelayedPopup</enum>
     </property>
     <property name="autoRaise">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="buttonToggleHappy">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>21</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>21</width>
       <height>25</height>
      </size>
     </property>
     <property name="popupMode">
      <enum>QToolButton::DelayedPopup</enum>
     </property>
     <property name="autoRaise">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="buttonNextSide">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>21</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>21</width>
       <height>25</height>
      </size>
     </property>
     <property name="popupMode">
      <enum>QToolButton::DelayedPopup</enum>
     </property>
     <property name="autoRaise">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="buttonToolDisk">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>21</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>21</width>
       <height>25</height>
      </size>
     </property>
     <property name="popupMode">
      <enum>QToolButton::DelayedPopup</enum>
     </property>
     <property name="autoRaise">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="buttonToggleOSB">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>21</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>21</width>
       <height>25</height>
      </size>
     </property>
     <property name="popupMode">
      <enum>QToolButton::DelayedPopup</enum>
     </property>
     <property name="autoRaise">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QLabel" name="labelFileName">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelImageProperties">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>128</red>
             <green>128</green>
             <blue>128</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Mid">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>157</red>
             <green>155</green>
             <blue>144</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>128</red>
             <green>128</green>
             <blue>128</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Mid">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>157</red>
             <green>155</green>
             <blue>144</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>118</red>
             <green>116</green>
             <blue>108</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Mid">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>157</red>
             <green>155</green>
             <blue>144</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QToolButton" name="buttonEject">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>21</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>21</width>
       <height>25</height>
      </size>
     </property>
     <property name="popupMode">
      <enum>QToolButton::DelayedPopup</enum>
     </property>
     <property name="autoRaise">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionMountDisk">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/silk-icons/icons/disk.png</normaloff>:/icons/silk-icons/icons/disk.png</iconset>
   </property>
   <property name="text">
    <string>Mount disk image...</string>
   </property>
   <property name="toolTip">
    <string>Mount a disk image</string>
   </property>
   <property name="statusTip">
    <string>Mount a disk image to D%1</string>
   </property>
  </action>
  <action name="actionMountFolder">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/silk-icons/icons/folder.png</normaloff>:/icons/silk-icons/icons/folder.png</iconset>
   </property>
   <property name="text">
    <string>Mount folder image...</string>
   </property>
   <property name="toolTip">
    <string>Mount a folder image</string>
   </property>
   <property name="statusTip">
    <string>Mount a folder image to D%1</string>
   </property>
  </action>
  <action name="actionEject">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/silk-icons/icons/control_eject.png</normaloff>:/icons/silk-icons/icons/control_eject.png</iconset>
   </property>
   <property name="text">
    <string>Unmount</string>
   </property>
   <property name="toolTip">
    <string>Unmount</string>
   </property>
   <property name="statusTip">
    <string>Unmount D%1</string>
   </property>
  </action>
  <action name="actionWriteProtect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/silk-icons/icons/lock_open.png</normaloff>
     <normalon>:/icons/silk-icons/icons/lock.png</normalon>:/icons/silk-icons/icons/lock_open.png</iconset>
   </property>
   <property name="text">
    <string>Write protected</string>
   </property>
   <property name="toolTip">
    <string>Toggle write protection</string>
   </property>
   <property name="statusTip">
    <string>Toggle write protection for D%1</string>
   </property>
  </action>
  <action name="actionEditDisk">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/silk-icons/icons/folder_edit.png</normaloff>:/icons/silk-icons/icons/folder_edit.png</iconset>
   </property>
   <property name="text">
    <string>Explore..</string>
   </property>
   <property name="toolTip">
    <string>Show properties</string>
   </property>
   <property name="statusTip">
    <string>Show D%1's properties</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/silk-icons/icons/drive_save.png</normaloff>:/icons/silk-icons/icons/drive_save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save image</string>
   </property>
   <property name="statusTip">
    <string>Save D%1</string>
   </property>
  </action>
  <action name="actionRevert">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/silk-icons/icons/arrow_undo.png</normaloff>:/icons/silk-icons/icons/arrow_undo.png</iconset>
   </property>
   <property name="text">
    <string>Revert to original</string>
   </property>
   <property name="toolTip">
    <string>Revert image to its last saved state</string>
   </property>
   <property name="statusTip">
    <string>Revert D%1 to its last saved state</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/silk-icons/icons/drive_rename.png</normaloff>:/icons/silk-icons/icons/drive_rename.png</iconset>
   </property>
   <property name="text">
    <string>Save as...</string>
   </property>
   <property name="toolTip">
    <string>Save to a file</string>
   </property>
   <property name="statusTip">
    <string>Save D%1 to a file</string>
   </property>
  </action>
  <action name="actionAutoSave">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/silk-icons/icons/drive_go.png</normaloff>:/icons/silk-icons/icons/drive_go.png</iconset>
   </property>
   <property name="text">
    <string>Auto save</string>
   </property>
   <property name="toolTip">
    <string>Commit changes to this disk automatically</string>
   </property>
   <property name="statusTip">
    <string>Toggle Auto Commit ON/OFF</string>
   </property>
  </action>
  <action name="actionBootOption">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/oxygen-icons/16x16/actions/flag_green.png</normaloff>:/icons/oxygen-icons/16x16/actions/flag_green.png</iconset>
   </property>
   <property name="text">
    <string>Folder Boot Options</string>
   </property>
   <property name="toolTip">
    <string>Change Boot Options</string>
   </property>
  </action>
  <action name="actionToggleChip">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/silk-icons/icons/package_delete.png</normaloff>
     <normalon>:/icons/silk-icons/icons/package_go.png</normalon>:/icons/silk-icons/icons/package_delete.png</iconset>
   </property>
   <property name="text">
    <string>Toggle Chip/Super Archiver compatibility</string>
   </property>
   <property name="toolTip">
    <string>Enable or disable Chip/Super Archiver compatibility. When enabled, the Chip is open</string>
   </property>
  </action>
  <action name="actionToggleHappy">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/silk-icons/icons/emoticon_unhappy.png</normaloff>
     <normalon>:/icons/silk-icons/icons/emoticon_happy.png</normalon>:/icons/silk-icons/icons/emoticon_unhappy.png</iconset>
   </property>
   <property name="text">
    <string>Toggle Happy compatibility</string>
   </property>
   <property name="toolTip">
    <string>Enable or disable Happy compatibility. When enabled, the drive is in Happy mode</string>
   </property>
  </action>
  <action name="actionNextSide">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/oxygen-icons/16x16/actions/2rightarrow.png</normaloff>:/icons/oxygen-icons/16x16/actions/2rightarrow.png</iconset>
   </property>
   <property name="text">
    <string>Load next software disk or side</string>
   </property>
   <property name="toolTip">
    <string>Load next software disk or side</string>
   </property>
  </action>
  <action name="actionToolDisk">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/silk-icons/icons/shield.png</normaloff>
     <normalon>:/icons/silk-icons/icons/shield_go.png</normalon>:/icons/silk-icons/icons/shield.png</iconset>
   </property>
   <property name="text">
    <string>Boot tool disk image</string>
   </property>
   <property name="toolTip">
    <string>Boot tool disk image the next time the Atari is powered on</string>
   </property>
  </action>
  <action name="actionToggleOSB">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/other-icons/toggleosb.png</normaloff>:/icons/other-icons/toggleosb.png</iconset>
   </property>
   <property name="text">
    <string>Boot translator disk image</string>
   </property>
   <property name="toolTip">
    <string>Boot translator disk image the next time the Atari is powered on</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
